src/textual/app.py

```diff
@@ -106,6 +109,8 @@ def __init__(
             driver_class (Type[Driver], optional): Driver class, or None to use default. Defaults to None.
             title (str, optional): Title of the application. Defaults to "Textual Application".
         """
+        App._init_uvloop()
```

Comment:
We have to initialise `uvloop` as soon as possible, as it will replace the event loop if it's installed - hence this initialisation code I had to add in the constructor of our Textual apps :confused: 

That's because this class is a child of the MessagePump one, which does `self._message_queue: PriorityQueue[MessagePriority] = PriorityQueue()` in its constructor.
...And this class inherits its constructor from its parent Queue, which contains the following code in Python versions 3.7, 3.8 and 3.9 - but no longer in 3.10:
```py
        if loop is None:
            self._loop = events.get_event_loop()
        else:
            self._loop = loop
```
( https://github.com/python/cpython/blob/3.9/Lib/asyncio/queues.py )

So as soon as our App is instantiated, these queues are created and they have a reference to a asyncio loop.
Which make them crash with Python versions older than 3.10.